apps/omg_watcher_info/test/omg_watcher_info/db/repo_test.exs

```diff
@@ -25,6 +25,15 @@ defmodule OMG.WatcherInfo.DB.RepoTest do
   require Utxo
 
   describe "DB.Repo.insert_all_chunked/3" do
+    # The current number of columns on the transaction table allow up to 8190
```

Comment:
Can you also describe how to derive the number 8190? So we know how to re-calculate.

Reply:
I added a test that both shows how to derive the number and that the number is correct (there was a typo in the comment, the actual max is 8191).